lang/python/python-pyxdg/Makefile

```diff
@@ -0,0 +1,35 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pyxdg
+PKG_VERSION:=0.26
+PKG_RELEASE:=1
+PKG_LICENSE:=LGPL-2.0
```

Comment:
If you're willing to maintain this, please state so. Also, please rearrange the variables similar to https://github.com/openwrt/packages/blob/master/utils/btrfs-progs/Makefile for example.